src/core/libmaven/mzSample.h

```diff
@@ -217,6 +217,15 @@ class mzSample
     * @brief Load sample (supported formats: .mzxml, .mzml, .mzdata, .mzcsv and .cdf)
     * @param filename Sample file name
     */
+
+    enum SampleType: int {
+        NONE = 0,
+        MS,
+        SRM,
+        MRM,
+        PRM
+    };
```

Comment:
Lets change `PRM` type to `DDA` (Data Dependent Acquisition) for PRM samples, and add a `DIA` (Data Independent Acquisition) for swath samples here. Suggested by @shubhra-agrawal.